Makefile

```diff
@@ -35,7 +35,7 @@ clean:
 $(RUNTIME_TOOLS_LINK):
 	ln -sf $(CURDIR) $(RUNTIME_TOOLS_LINK)
 
-.PHONY: test .gofmt .govet .golint
```

Comment:
> Putting them into .PHONY is a way to defend in case of there is .gofmt, .govet and so son which will break make process.

And it's also a way of saying “these target names are just for convenience.  We are not attempting to create files called `.gofmt`, etc.”.  I'd rather leave them `.PHONY` so folks don't get confused (“Why is this recipe not creating a `.gofmt` file?”).